deposit_addr.go

```diff
@@ -44,66 +36,63 @@ func depositHasher() Sha256 {
 	return h
 }
 
-// EvmDepositTweak Compute the tweak bytes for an EVM deposit address.
+// DepositTweak Compute the tweak bytes for a deposit address.
 //
-// This is defined as
+// This is generally defined as
 //
-//	taggedHash( AuxData || EvmTag || ChainId || LBTCAddress || WalletAddress )
+//	taggedHash( AuxData || ChainId || LBTCAddress || WalletAddress )
```

Comment:
1) by the fact the byte is still there
2) mb `LChainId` is better to show that it's something custom
```suggestion
//	taggedHash( AuxData || DeprecatedChainTag || LChainId || LBTCAddress || WalletAddress )
```